### PR TITLE
Add content for quarto website workshop

### PR DESCRIPTION
**Pull request by KristinaRiemer**

### CODE REVIEW

File: 202308-quarto-websites/quarto-websites-notes.qmd [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+    -   `.Rproj` is for project
+    -   `_quarto.yml` is config file
+        -   Specify metadata about website and its different pages
+        -   Customize navigation, page layout, and theme
+    -   Two `.qmd` files
+        -   `about.qmd` & `index.qmd`
+        -   These are pages on the website
+-   See what website looks like
+    -   Open any `.qmd` file
+    -   Use "Render" button, just like with Quarto files
```
Comment (Aariq) [inline 202308-quarto-websites/quarto-websites-notes.qmd:114]: There's also a"Render Webiste" button in the "Build" tab—not sure if that does something different but might be worth investigating

File: 202308-quarto-websites/quarto-websites-notes.qmd [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+    -   Add blog with posts
+        -   Nice for sharing code examples or outputs
+        -   Not actually going to go through this
+        -   [How to](https://samanthacsik.github.io/posts/2022-10-24-quarto-blogs/)
+    -   Do more fancy things with CSS
+        -   Beyond scope of this workshop
+-   Get inspiration from some great Quarto websites!
+    -   [Eric](https://ericrscott.com/)
+    -   [Maya Gans](https://maya.rbind.io/)
+    -   [Heidi Steiner](https://heidiesteiner.netlify.app/)
```
Comment (Aariq) [inline 202308-quarto-websites/quarto-websites-notes.qmd:184]: If you send this out in a followup email it'd be useful to link directly the corresponding github repos as well

**Review by Aariq — APPROVED** [review]
Looks good!  I think you've made good choices about what to go into detail with and what to show but not do.

File: 202308-quarto-websites/quarto-websites-notes.qmd (hunk 1 above)
Comment (KristinaRiemer) [reply 202308-quarto-websites/quarto-websites-notes.qmd:114]: Oh, awesome! I didn't know that was there. And it didn't seem great to have to open up a `.qmd` just to render it locally

**Pull request merged**